Updates docs to the newest connector version (#43)
conduitio-labs/conduit-connector-kinesis: README.md

| `aws.accessKeyId`     | Access Key ID associated with your AWS resources | true     | ""            |
| `aws.secretAccessKey` | Secret Access Key associated with your AWS resources | true     | ""            |
| `aws.region`     | Region associated with your AWS resources | true     | ""            |

> | `aws.url`     | (LOCAL TESTING ONLY) the url override to test with localstack | false     | ""            |

Reviewer: Not sure I understand what `LOCAL TESTING ONLY` is about. Shouldn't we have a default value that works locally, and this configuration still needed? (Maybe, there's something I'm missing!)

Author: Hello @raulb 

aws.url forces the kinesis sdk to point to localstack, which is a docker image that simulates aws kinesis. I agree that it is not ideal having the user docs include that parameter, this is more of a "useful for contributors only" parameter type.

I'm thinking of excluding the aws.url from the parameters and just add it as such:

```go
func TestThings(t *testing.T) {
	d := Destination{}
	is := is.New(t)
	err := d.Configure(context.Background(), map[string]string{
		"some-random-test-parameter": "some-url",
	})
	is.NoErr(err)
	// ...
}
```

And then check for the key existence in the .Configure method.

What do you think? how do you guys normally handle test only parameters? Do you include them at the parameters list? Or would you do something like I described?

Reviewer 2: I don't think it hurts, if someone wants to play around with the connector locally 🤷 I don't think we need to ALL CAPS the warning the way it is, Maybe just a mention at the end if you ask me.
```suggestion
| `aws.url`     | The URL for AWS (useful when testing the connector with localstack). | false     | ""            |
```

Reviewer: @alarbada got it. Thanks for the explanation. I think in that case it's fine to leave it as a parameter. Agreed with @lovromazgon 's suggestion to improve its formatting. All caps seemed like a placeholder rather than something finalized.
